simulator/infra/memory/t/unit_test.cpp

```diff
@@ -124,6 +124,46 @@ TEST_CASE( "Func_memory: Write_Read_Not_Initialized_Mem_Test")
     CHECK( func_mem.read<uint16>( write_addr + 2) == right_ret);
 }
 
+TEST_CASE( "Func_memory: Host_Guest_Memcpy")
```

Comment:
There are at least three different unit tests, and they have to be separated to different test cases